maven-plugin/pom.xml

```diff
@@ -113,7 +113,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-source-plugin</artifactId>
-            <version>3.0.0</version>
+            <version>3.0.1-SNAPSHOT</version>
```

Comment:
Hey just curious if this line change was intentional? Trying to push out a PR for a 3.0.2-SNAPSHOT and not sure if this is supposed to be updated or not.

Reply:
that looks like an error ... (much later he says)